cypress/integration/pages/photoGalleryPage/tests.js

```diff
@@ -0,0 +1,63 @@
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    it('should render a H1, which displays the headline', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          cy.get('h1').should('contain', body.promo.headlines.headline);
+        },
+      );
+    });
+
+    it('should render a timestamp', () => {
+      cy.request(
+        `${config[service].pageTypes.photoGalleryPage.path}.json`,
+      ).then(({ body }) => {
+        const { lastPublished, firstPublished } = body.metadata;
+        const timeDifferenceMinutes =
+          (lastPublished - firstPublished) / 1000 / 60;
+        const minutesTolerance = 1;
+        cy.get('time')
+          .eq(0)
```

Comment:
Look at the docs for `get` not really seeing an a situation where it would return `0`: https://docs.cypress.io/api/commands/get.html#Syntax

Reply:
See above comment about `.eq(0)`.

Reply:
Oh, so `eq` gets by index: https://docs.cypress.io/api/commands/eq.html#Syntax !

That makes sense now, I assumed it was some kind of assertion, i.e. equals 0. I'm still unsure why the api is `eq`!